### ISSUE TITLE
Weird issue with using Tinkers Construct tables as requirements (may be BQ bug)

### ISSUE THREAD

**Issue by brisingraerowing** [fonnymunkey/BQTweaker#2]
If I add, say, the Tinkers Stencil Table as a quest requirement, it will show the name "Crafting Station" instead of "Stencil Table". I'm not sure if this is a bug with BQ or this mod. I'll do a quick test with just BQ and Tinkers to see.

**Comment by brisingraerowing** [comment]
I've found that adding a regular Crafting Station before the Stencil Table makes the name show up normally, then I can remove the Crafting Station. I have Simple Crafting Station, and am using that as a requirement instead of the normal Crafting Station.

**Comment by fonnymunkey** [comment]
What kind of quest requirement, retrieval task? I'm not sure why BQTweaker would affect what name shows up, as it doesn't replace the task panel, just renders them in the same page.

**Comment by brisingraerowing** [comment]
Yup, retrieval task. I'm not sure why it's broken either. At least there's a workaround.

**Comment by fonnymunkey** [comment]
Ah, I think I see what happened.

Tinker's uses the same item id for it's crafting tables, and differentiates them using metadata. When selecting the item for the quest, did you hit the "*" next to the "OreDict" section? This tells BQ to accept any item with that id, and to ignore metadata. As such, BQ just displays the name of whatever item is default (aka, metadata 0), which in this case is the Crafting Station.

To test, create one retrieval task with the Stencil Table, making sure not to click the "*", then create a second task specifically clicking the "*". The task you did not click the "*" will say Stencil Table and show the same item, but the task you did click "*" will say Crafting Station and will be cycling through all the different tables.
